.github/workflows/.dco.yml

```diff
@@ -35,14 +35,14 @@ jobs:
               return context.payload.pull_request.base.ref;
             }
             return context.ref.replace(/^refs\/heads\//g, '');
-      -
-        name: Validate
-        run: |
-          docker run --rm \
-            -v "$(pwd):/workspace" \
-            -e VALIDATE_REPO \
-            -e VALIDATE_BRANCH \
-            alpine:${{ env.ALPINE_VERSION }} sh -c 'apk add --no-cache -q bash git openssh-client && git config --system --add safe.directory /workspace && cd /workspace && hack/validate/dco'
-        env:
-          VALIDATE_REPO: ${{ github.server_url }}/${{ github.repository }}.git
-          VALIDATE_BRANCH: ${{ steps.base-ref.outputs.result }}
+#      -
+#        name: Validate
+#        run: |
+#          docker run --rm \
+#            -v "$(pwd):/workspace" \
+#            -e VALIDATE_REPO \
+#            -e VALIDATE_BRANCH \
+#            alpine:${{ env.ALPINE_VERSION }} sh -c 'apk add --no-cache -q bash git openssh-client && git config --system --add safe.directory /workspace && cd /workspace && hack/validate/dco'
+#        env:
+#          VALIDATE_REPO: ${{ github.server_url }}/${{ github.repository }}.git
+#          VALIDATE_BRANCH: ${{ steps.base-ref.outputs.result }}
```

Comment:
Temporarily disabled the DCO check here (this must be reverted after merging, or we can drop this commit before merging (once CI passes))

Reply:
- opened https://github.com/moby/moby/pull/45820 to re-enable it